test/cpp/end2end/end2end_test.cc

```diff
@@ -218,8 +226,39 @@ class TestServiceImpl : public ::grpc::testing::EchoTestService::Service {
   explicit TestServiceImpl(const grpc::string& host)
```

Comment:
Sorry I was talking about the implementation of this class was moved to a separate file: https://github.com/grpc/grpc/blob/master/test/cpp/end2end/test_service_impl.cc. Is it possible to make the new changes there and use that?